client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx

```diff
@@ -64,49 +52,44 @@ const WooConfirm: Step = function WooCommerceConfirm( { navigation } ) {
 		requestLatestAtomicTransfer( siteId );
 	}, [ requestLatestAtomicTransfer, siteId ] );
```

Comment:
This would be also sooo elegant as a React Query 🙂 Fetch from the `/latest` endpoint, but specify we don't want to cache the data and always want to fetch fresh info.